portfolio/src/main/webapp/script.js

```diff
@@ -55,6 +55,13 @@ function createRecommendationElement(toDo) {
   comment.innerHTML = toDo.comment; 
   recommendation.appendChild(comment);
 
+  if (toDo.imageUrl !== undefined){
```

Comment:
`if (!toDo.imageUrl) {` ?

Reply:
ah.. just tried- that doesn't work. I think because the value of toDo.imageUrl in datastore is = undefined?

Reply:
Oops sorry I meant
`if (toDo.imageUrl)`

Reply:
oh right! makes sense!